keel-igor/src/main/kotlin/com/netflix/spinnaker/keel/igor/DeliveryConfigImporter.kt

```diff
@@ -10,7 +9,6 @@ import org.springframework.stereotype.Component
 /**
  * Provides functionality to import delivery config manifests from source control repositories (via igor).
  */
```

Comment:
this is initialized as a bean in the config, it doesn't need the component annotation. Previously we weren't even scanning these packages for components. I fixed that also.

Reply:
Maybe we should stick with one of those approaches, though? Consider keeping the `@Component` annotation and removing the `@Bean` factory function.

Reply:
Considering the `@Bean` approach is the one that worked for that class, I didn't want to make an unrelated change in my PR. We will need both methods anyways since the `@Bean` one is used for creating retrofit clients